aws/aws-how-to/instances/create-ec2-using-ubuntu-image.rst

```diff
@@ -0,0 +1,118 @@
+* Access Credentials: There are three types: access keys, X.509 certificates and key pairs. The first and second type allow you to connect to the Amazon APIs. Which type of credential depends on which API and tool you are using. Some APIs and tools support both options, whereas others support just one. The third type is SSH public/private key pairs that are used for initial logins to newly created instances.
+
+    * access keys: Symmetric key encryption. These are for making requests to AWS product REST or Query APIs. Can be obtained/regenerated from the Access Keys tab on the AWS Security Credentials page.
+    * X.509 certificates: Public key encryption. Use X.509 certificates to make secure SOAP protocol requests to AWS service APIs. These are the credentials you will use when using the command-line ec2 API tools. Can be obtained/regenerated from the X.509 Certificates tab on the AWS Security Credentials page.
+    * key pairs: SSH key pairs. When you create an instance, Amazon inserts the public key of your SSH key pair into your new instance so that you can log in using your private key. You can add new SSH key pairs through the AWS management console by clicking on Key Pairs under Networking and Security in the Navigation pane and then the Create Key Pair button. After specifying a name you will be prompted to download and save your private key. EC2 stores the public portion of your key pair, and inserts it into /home/ubuntu/.ssh/authorized_keys when you instantiate your instance. If you lose this private key, it cannot be downloaded again; you will need to regenerate a new key pair.
+
+To use EC2 services, you need to create credentials below.
+
+* Create a key pair and download the private key 
+
+    * Click Key Pairs under Networking and Security in the Navigation pane and then click the Create Key Pair button (save it in e.g. ~/.ec2/ec2.pem). This private key is for making SSH connections to newly created instances. 
+
+* Set up Amazon API credentials
+
+    * Go to Account at the right upper corner by clicking your name and click Security Credentials
+    * Go to Access keys section and create access key for your account
+    * Download access keys and save those information under ~/.ec2/credentials file
+    * Create `config` file under ~/.ec2/ and config your default region to use EC2 services like below
+
+.. code::
+
+    [default]
+    region = us-east-1
+
+
+Official Ubuntu Cloud Guest Amazon Machine Images (AMIs)
+--------------------------------------------------------
+
+The Official AMI Ids can be obtained from `Find Ubuntu images on AWS <https://canonical-aws.readthedocs-hosted.com/en/latest/aws-how-to/instances/find-ubuntu-images/>`_. Official Ubuntu AMIs are published by the 'Canonical' user, with Amazon ID '099720109477'. Images containing the string 'ubuntu' but not owned by that ID are not official AMIs.
```

Comment:
We don't have public 2P listings anymore. All 2P listings are limited currently and not visible to new users. But if we want to add that information to public doc, I will add marketplace account and user name for limited listings

Reply:
I would say, let's remove limited listing information from this site and our public page (Trusty listing) since listings should be shared and are not visible to public. Also, we are promoting customers to use EC2 console images for Pro.

Reply:
+1 to not specifically mentioning 2P listings for now.

@carlosbravoa could you weigh in on the last comment? I think this generally makes sense for this page. I think having a page on "how to request support for EOL releases" makes sense in a reference page later on.

Reply:
Give me some time. I can review it during this afternoon. Thanks

Reply:
This doc needs a complete overhaul, not just updates. The title is a how-to but the doc is mix of a tutorial and recipes. Have you discussed this with @k-dimple ? We could break it into smaller parts too

Reply:
@carlosbravoa Or I think we can just change topic how to start instance with AWS CLI? since this article covers more likely launching instances with cli. Then, it makes sense to explain all credential setup, finding right ubuntu image and API setup links?

Reply:
(thining out loud) Given that this is a starters guide, I would like to lead the reader to success (being able to log into a launched instance) as soon as possible. So try to keep this as short as possible and reference other pages where more information could be helpful.